.github/workflows/docker.yml

```
@@ -0,0 +1,36 @@
+        run: echo "TAG=$(date +%Y-%m-%d)-${GITHUB_SHA::7}" >> $GITHUB_ENV
+
+      - name: Build and push Docker image
+        uses: docker/build-push-action@v5
+        with:
+          context: .
+          push: true
+          tags: |
+            crretoxmas2024.azurecr.io/ithaka-frontend:latest
+            crretoxmas2024.azurecr.io/ithaka-frontend:${{ env.TAG }}
+          cache-from: type=registry,ref=crretoxmas2024.azurecr.io/ithaka-frontend:cache
+          cache-to:   type=registry,ref=crretoxmas2024.azurecr.io/ithaka-frontend:cache,mode=max
```

YAMLlint (1.37.1)

[error] too many spaces after colon

(colons)